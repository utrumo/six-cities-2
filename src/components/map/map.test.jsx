import React from 'react';
import renderer from 'react-test-renderer';
import Map from './map.jsx';

const view = {
  latitude: 52.370216,
  longitude: 4.895168,
  zoom: 12
};

const markers = [
  {
    id: 10,
    latitude: 52.3909553943508,
    longitude: 4.85309666406198
  },
  {
    id: 11,
    latitude: 52.369553943508,
    longitude: 4.85309666406198
  }
];

it(`Map renders correctly`, () => {
  const tree = renderer.create(<Map view={view} markers={markers}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
