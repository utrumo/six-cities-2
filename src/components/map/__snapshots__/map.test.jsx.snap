// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map component renders correctly after restart 1`] = `
<div
  id="map"
  style={
    Object {
      "height": "100%",
    }
  }
/>
`;
